features/user_can_use_the_navbar.feature

```diff
@@ -0,0 +1,16 @@
+Feature: User can see a sidebar
```

Comment:
Your file says `User can see a sidebar` but the title of your feature is `User can see a sidebar`? 🤔
Try to keep some consistency with that.